src/components/EditPartnership/EditPartnership.tsx

```diff
@@ -0,0 +1,115 @@
+import { Box, Button } from '@material-ui/core';
+import React, { FC } from 'react';
+import { Except } from 'type-fest';
+import { PartnershipType, UpdatePartnership } from '../../api';
+import { DialogForm, DialogFormProps } from '../Dialog/DialogForm';
+import { DialogState } from '../Dialog/useDialog';
+import {
+  CheckboxesField,
+  CheckboxOption,
+  RadioField,
+  RadioOption,
+  SubmitError,
+} from '../form';
+import { PartnershipCardFragment } from '../PartnershipCard/PartnershipCard.generated';
+import {
+  useDeletePartnershipMutation,
+  useUpdatePartnershipMutation,
+} from './EditPartnership.generated';
+
+interface EditPartnershipProps {
+  state: DialogState;
```

Comment:
This should just forward on the `DialogForm` props. The `useDialog` props are a subset of this, but this component shouldn't care about that (how it's used).

See what I did with `EditOrganization`:
https://github.com/SeedCompany/cord-field/blob/a42a18dbe55ddf3f5933162662b0ca3e863f5587/src/scenes/Organizations/Edit/EditOrganization.tsx#L12-L17

Reply:
Fixed